src/CMakeLists.txt

```diff
@@ -763,6 +763,11 @@ else()
 		set(RELEASE_WARNING_FLAGS "")
 	endif()
 
+	if (NOT APPLE AND UNIX AND USE_LUAJIT)
+		# Make symbols that start with "mtffi_" available to LuaJIT's FFI library.
+		SET(CMAKE_EXE_LINKER_FLAGS  "${CMAKE_EXE_LINKER_FLAGS} -Wl,--dynamic-list=${CMAKE_SOURCE_DIR}/src/ffi_dynamic_list.txt")
```

Comment:
this should use [`target_link_options`](https://cmake.org/cmake/help/latest/command/target_link_options.html) on mt & mtserver since users I think can override `CMAKE_EXE_LINKER_FLAGS` entirely